common/ebulletin.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
This should probably live under `controllers/ebulletin/lib/newsletter-service` as we only use it in that router right now